lib/src/test/java/uk/gov/di/ipv/core/library/domain/gpg45/Gpg45ProfileEvaluatorTest.java

```diff
@@ -588,4 +612,64 @@ void parseGpg45ScoresFromCredentialsShouldReturnCredentialItemsMapForADcmawCrede
         assertEquals(2, dcmawItems.get(0).getEvidenceScore().validity());
         assertEquals(1, dcmawItems.get(0).getActivityHistoryScore());
     }
+
+    @Test
+    void shouldCallCIStorageSystemToGetCIs() throws Exception {
+        Map<CredentialEvidenceItem.EvidenceType, List<CredentialEvidenceItem>> evidenceMap =
```

Comment:
Could this be moved to the setup? If it's the same across both tests? Mainly for readability.